foundation/foundation-upgrade/src/test/resources/specifications/v1_7_0/enable-habushu-build-cache-migration.feature

```diff
@@ -0,0 +1,17 @@
+@habushu-build-cache-migration
+Feature: Adds the <build><directory>dist</directory></build> tag to the pom file if not present.
+
+  Scenario: Migrate a pom file
```

Comment:
S: again, helpful if you directly say what it is here instead of making someone infer it from the rest.

Reply:
Reasonable.  Done.